release-calendar/src/client/models/view-models.ts

```diff
@@ -14,21 +14,28 @@
  * limitations under the License.
  */
 
-import {Channel, Promotion, Release as ReleaseEntity} from '../../types';
+import {Channel, Promotion} from '../../types';
 
-export class Release {
-  constructor(entity: ReleaseEntity) {
-    const currentPromotion = entity.promotions[0];
-    this.name = entity.name;
-    this.channel = currentPromotion.channel;
-    this.date = currentPromotion.date;
-    this.isRollback = this.channel == Channel.ROLLBACK;
+export class EventInput {
```

Comment:
Since FullCalendar has an `EventInput` type already, can we rename this to something else?